api/RTCPeerConnection.json

```diff
@@ -1878,6 +1880,56 @@
           }
         }
       },
+      "icecandidateerror_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/icecandidateerror_event",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#event-icecandidateerror",
+          "description": "<code>icecandidateerror</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
The existing data for `onicecandidateerror` is wrong. This event was implemented in Chromium 77:
https://storage.googleapis.com/chromium-find-releases-static/7d9.html#7d952183f841e90ef48c36c6f29ab896e7175934
https://www.chromestatus.com/feature/6602864602382336

The event also has an interface, and https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnectionIceErrorEvent#browser_compatibility already looks correct. Can you copy that data?

Then, can you mirror this to the other Chromium-based browsers?